internal/cloudentity/client.go

```diff
@@ -0,0 +1,135 @@
+package cloudentity
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+var (
+	HeaderAuthorizerURL = "X-Kusk-Authorizer-URL"
+	HeaderAPIGroup      = "X-Kusk-API-Group"
+)
+
+type Client struct {
+	hc  http.Client
+	url string
+}
+
+func New(url string) *Client {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	return &Client{
+		hc:  http.Client{Timeout: 1 * time.Minute, Transport: tr},
+		url: url,
+	}
+}
+
+func (c *Client) PutAPIGroups(ctx context.Context, req *APIGroupsRequest) error {
+	b, err := json.Marshal(req)
+	if err != nil {
+		return fmt.Errorf("marshal json: %w", err)
+	}
+
+	httpReq, err := http.NewRequestWithContext(ctx, http.MethodPut, c.url+"/apis", bytes.NewBuffer(b))
+	if err != nil {
+		return fmt.Errorf("http new request: %w", err)
+	}
+
+	httpReq.Header.Set("Content-Type", "application/json")
```

Comment:
You repeat `"Content-Type"` and `"application/json"`. I think it might be better to define these as constants in a `httpcommon` package or something.